<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_MakeParticle</class>
 <widget class="QDialog" name="DLG_MakeParticle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make particles</string>
  </property>
  <widget class="QTabWidget" name="TB_Method">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="T_Cube">
    <attribute name="title">
     <string>Cube</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="L_CUBE_NX">
        <property name="text">
         <string>Nx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_CUBE_NX"/>
      </item>
      <item>
       <widget class="QLabel" name="L_CUBE_NY">
        <property name="text">
         <string>Ny</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_CUBE_NY"/>
      </item>
      <item>
       <widget class="QLabel" name="L_CUBE_NZ">
        <property name="text">
         <string>Nz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_CUBE_NZ"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>351</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="L_CUBE_LOC">
        <property name="text">
         <string>Start loc.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_CUBE_LOC"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="T_Plane">
    <attribute name="title">
     <string>Plane</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>351</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="L_PLANE_LOC">
        <property name="text">
         <string>Start loc.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_PLANE_LOC"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="L_PLANE_NX">
        <property name="text">
         <string>Nx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_PLANE_NX"/>
      </item>
      <item>
       <widget class="QLabel" name="L_PLANE_NZ">
        <property name="text">
         <string>Nz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_PLANE_NZ"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="L_PLANE_DIR">
        <property name="text">
         <string>Dir.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_PLANE_DIR"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="T_Circle">
    <attribute name="title">
     <string>Circle</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>38</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="L_CircleLocation">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_CircleLocation"/>
      </item>
      <item>
       <widget class="QLabel" name="L_CircleDirection">
        <property name="text">
         <string>Direction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_CircleDirection"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="L_CircleDiameter">
        <property name="text">
         <string>Diameter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_CircleDiameter"/>
      </item>
      <item>
       <widget class="QLabel" name="L_NumParticle">
        <property name="text">
         <string>Num. particles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_NumParticle"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>371</width>
     <height>106</height>
    </rect>
   </property>
   <property name="title">
    <string>Particle property</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="L_MIN_RADIUS">
       <property name="text">
        <string>Min. radius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_MIN_RADIUS"/>
     </item>
     <item>
      <widget class="QLabel" name="L_MAX_RADIUS">
       <property name="text">
        <string>Max. radius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_MAX_RADIUS"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>351</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="L_P_SPACING">
       <property name="text">
        <string>Particle spacing</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LE_P_SPACING"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="L_NUM_PARTICLE">
       <property name="text">
        <string>Num. particle</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LE_NUM_PARTICLE">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GB_MaterialProperty">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>284</y>
     <width>371</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Material property</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>351</width>
      <height>126</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="L_Type">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="CB_Type"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="L_YoungsModulus">
       <property name="text">
        <string>Youngs modulus</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LE_Youngs"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="L_Density">
       <property name="text">
        <string>Density</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LE_Density"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="L_PoissonRatio">
       <property name="text">
        <string>Poisson ratio</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="LE_PoissonRatio"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="L_ShearModulus">
       <property name="text">
        <string>Shear modulus</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="LE_ShearModulus"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="PB_Ok">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_Cancle">
      <property name="text">
       <string>Cancle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="GB_BASIC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>53</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="L_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>21</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GB_RealTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Real time creating</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="L_NumParticlesPer">
       <property name="text">
        <string>The number of particles(per second)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_NumParclesPer"/>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="RB_OneByOne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>One by one drop</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_OneByGroup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>One by group drop</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LE_Name</tabstop>
  <tabstop>TB_Method</tabstop>
  <tabstop>LE_PLANE_NX</tabstop>
  <tabstop>LE_PLANE_NZ</tabstop>
  <tabstop>LE_PLANE_DIR</tabstop>
  <tabstop>LE_PLANE_LOC</tabstop>
  <tabstop>LE_MIN_RADIUS</tabstop>
  <tabstop>LE_MAX_RADIUS</tabstop>
  <tabstop>LE_P_SPACING</tabstop>
  <tabstop>LE_NUM_PARTICLE</tabstop>
  <tabstop>CB_Type</tabstop>
  <tabstop>LE_Youngs</tabstop>
  <tabstop>LE_Density</tabstop>
  <tabstop>LE_PoissonRatio</tabstop>
  <tabstop>LE_ShearModulus</tabstop>
  <tabstop>PB_Ok</tabstop>
  <tabstop>PB_Cancle</tabstop>
  <tabstop>LE_CUBE_NX</tabstop>
  <tabstop>LE_CUBE_NY</tabstop>
  <tabstop>LE_CUBE_LOC</tabstop>
  <tabstop>LE_CUBE_NZ</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
